src/StateService/StatePlugin.cs

```diff
@@ -15,27 +19,52 @@ public class StatePlugin : Plugin, IPersistencePlugin
         public override string Name => "StateService";
         public override string Description => "Enables MPT for the node";
 
-        private IActorRef store;
+        internal IActorRef Store;
+        internal IActorRef Validator;
```

Comment:
I think this also should have another name instead of `Validator`
It is acting like a `MPTDispacher`, or  `MPTAgent`

Reply:
Let's call it `Dispacher`.

Reply:
Done.